kinto_wizard/yaml2kinto.py

```diff
@@ -14,7 +14,7 @@
             async_client,
             bucket=bucket,
             collection=collection,
-            records=delete_missing_records
+            records=True
```

Comment:
What is this change about? From context I'm guessing it's meant to increase test coverage, but I don't get it.

Reply:
Actually it is a bug that I fixed because even if delete_missing_records is false we still want to grab the records if we inspect the server we want to be able to make a diff to choose if we want to update the record or overwrite it.

Reply:
That's what tests are for right? I guess the usecase for records=False is when force is true, which is already covered by tests.

Reply:
> That's what tests are for right? 

Not really, tests won't assert that every path in the code call it with records=True

>  I guess the usecase for records=False is when force is true, which is already covered by tests.

Apparently it's for when dump without --full or --records